packages/components/src/components/popover-message-checkbox/popover-message-checkbox.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Checkbox from '../checkbox';
+
+type TPopoverMessageCheckbox = {
+    defaultChecked?: boolean;
+    message: string;
+    name?: string;
+    onChange: React.FormEventHandler<HTMLInputElement> &
+        ((e: React.ChangeEvent<HTMLInputElement> | React.KeyboardEvent<HTMLSpanElement>) => void);
+    checkboxLabel: string;
+};
+
+const PopoverMessageCheckbox = ({
+    checkboxLabel,
+    defaultChecked,
+    onChange,
+    message,
+    name,
+}: TPopoverMessageCheckbox) => (
+    <React.Fragment>
+        {message}
+        <Checkbox defaultChecked={defaultChecked} onChange={onChange} name={name} label={checkboxLabel} />
```

Comment:
Type of label in CheckBox Component is marked as string. But shouldnt we change it to ReactNode since <Localize ../>can be passed as label?  
If then checkboxLabel should also be type of ReactNode. right? Correct me if i am wrong.

Reply:
Will make changes in Checkbox component